users/models.py

```diff
@@ -149,6 +157,10 @@ def full_legal_name(self) -> str:
             return " ".join([self.first_name, self.last_name])
         return ""
 
+    @property
+    def preferred_name(self) -> str:
+        return self.preferred_first_name if self.preferred_first_name else self.first_name
```

Comment:
Err wait I remember being a bit confused by this--the name `preferred_name` makes it sound as though this might be the preferred _full_ name of the user rather than the preferred _first_ name of the user (the implementation suggests it's the latter), what do you think?